src/resources/items/index.tsx

```diff
@@ -17,13 +17,13 @@ const ItemCreate = (): React.ReactElement => {
 const ItemEdit = (): React.ReactElement => {
   return (
```

Comment:
@salony75way - can you teach me how this next fix works?Have we just changed the order of the attributes?